src/app/components/source-graphs/source-volume/source-volume.component.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../../../styles/common';
+$side-label-size: 44px;
+
+:host {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding-left: $side-label-size;
+  width: 100%;
+  height: 100%;
+  background-color: $cl-base-40;
+}
```

Comment:
It needs to be more consistent with the designs:
![screen shot 2017-09-21 at 12 05 23](https://user-images.githubusercontent.com/125300/30689061-88be8862-9ec8-11e7-8859-c7e103629e30.png)